### DISCUSSION TITLE
How to build package and run

### DISCUSSION THREAD

**Discussion by xushuhui** [golang/vscode-go#1624] [Help] [answered]
I do `Go: Build Current Package` , But not find exe file, and no run 
I want build package and auto run like Goland. What should I do?

**Comment by hyangah** [comment, marked as answer]
Run > Start Debugging (F5)
Run > Start Without Debugging (F5)
 
More fancy way of configuring task running is 
   https://github.com/golang/vscode-go/blob/master/docs/tasks.md

**Reply by gabrielpl** [reply]
The [documentation](https://github.com/golang/vscode-go/blob/master/docs/features.md#run-your-code) says: 

> Behind the scenes, the Debug: Start without Debugging command calls go run

But this command (Start Without Debugging (Ctrl + F5) is currently starting delve anyway.
What am I missing?

**Reply by suzmue** [reply]
The command starts delve, which acts as the debug adapter. Delve itself will run the program without debugging.
